src/components/Shell/_mixins.scss

```diff
@@ -101,24 +101,16 @@
   }
 
   &__skip-to-content {
-    display: flex;
     position: relative;
-    height: 0;
+    display: flex;
     justify-content: center;
 
-    &__link {
-      @include hidden;
-
+    &__link.#{$prefix}--skip-to-content:focus {
+      position: relative;
+      border-color: $focus;
+      background-color: $interactive-01;
       margin-top: $header__spacing__padding;
-      margin-left: -$toolbar__sizing__width;
-      transition: unset;
-      z-index: z($layer: floating);
-
-      &:focus {
-        width: auto;
-        padding: $button-padding;
-        clip: auto;
-      }
+      font-weight: 600;
```

Comment:
had to add this weight because as I was checking the component with DAP, the latest ruleset now flags it for now having enough contrast + weight:

<img width="592" alt="Screen Shot 2020-03-27 at 3 14 00 PM" src="https://user-images.githubusercontent.com/9057921/77797902-e24a6b80-703f-11ea-996f-8a4deefa00ba.png">